fedmsg_meta_fedora_infrastructure/bodhi.py

```diff
@@ -43,7 +43,24 @@ def get_sync_product(msg):
     return product
 
 
+def get_sync_release(msg):
```

Comment:
Please note that we have more release postfix letters: `f` for flatpack, and `c` for container.